SECURITY.md

```diff
@@ -0,0 +1,93 @@
+
+CVEs will be assigned to vulnerabilities. Due to the process and time it takes to obtain a CVE ID, disclosures will happen first. Once the disclosure is public the process will begin to obtain a CVE ID. Once the ID has been assigned the disclosure will be updated.
+
+If the vulnerability reporter would like their name and details shared as part of the disclosure process we are happy to. We will ask permission and for the way the reporter would like to be identified. We appreciate vulnerability reports and would like to credit reporters if they would like the credit.
+
+Vulnerability disclosures are published in the Security Advisories sections in each repository. The disclosures will contain an overview, details about the vulnerability, a fix for the vulnerability that will typically be an update, and optionally a workaround if one is available.
+
+Disclosures will be published on the same day as a release fixing the vulnerability after the release is published.
+
+Here are the links to security advisories for each repository:
+
+- [Notation CLI Security Advisories](https://github.com/notaryproject/notation/security/advisories)
+- [Notation-Go Library Security Advisories](https://github.com/notaryproject/notation-go/security/advisories)
+- [Notation-Core-Go Library Security Advisories](https://github.com/notaryproject/notation-go/security/advisories)
+- [Notation Spcecification Security Advisories](https://github.com/notaryproject/notaryproject/security/advisories)
+
+## Security Team Membership
+The security team is made up of a subset of the Notation project maintainers who are willing and able to respond to vulnerability reports.
+
+### Responsibilities
+- Members MUST be active project maintainers on active (non-deprecated) Notation projects as defined in the governance
+- Members SHOULD engage in each reported vulnerability, at a minimum to make sure it is being handled
+- Members MUST keep the vulnerability details private and only share on a need to know basis
+
+### Membership
+New members are required to be active maintainers of Notation projects who are willing to perform the responsibilities outlined above. The security team is a subset of the maintainers. Members can step down at any time and may join at any time.
+
+From time to time, Notation projects are deprecated. If at any time a security team member is found to be no longer be an active maintainer on active Notation projects, this individual will be removed from the security team.
+
+## Patch and Release Team
+When a vulnerability comes in and is acknowledged, a team - including maintainers of the Notation project affected - will be assembled to patch the vulnerability, release an update, and publish the vulnerability disclosure. This may expand beyond the security team as needed but will stay within the pool of Notation project maintainers.
```

Comment:
Here `Notation project` is mentioned, not `notary project` at the beginning of this doc. It's better to be consistent. Since we only cover `Notation` repos, my suggestion is to use `Notation project`.

Reply:
This Security Policy will apply to all sub-projects under Notary Project, including Notation. I fixed all references.